presto-native-execution/presto_cpp/main/TaskManager.h

```diff
@@ -64,13 +64,15 @@ class TaskManager {
           std::unordered_map<std::string, std::string>>&&
           connectorConfigStrings);
 
-  // Iterates through a map of resultRequests and fetches data from
-  // buffer manager. This method uses the getData() global call to fetch
-  // data for each resultRequest bufferManager. If the output buffer for task
-  // is not found, prepares the Result object with completed flags set to false
-  // and notifies the future.
-  // Note: This method is made public for unit testing purpose only.
+  /// Iterates through a map of resultRequests and fetches data from
+  /// buffer manager. This method uses the getData() global call to fetch
+  /// data for each resultRequest bufferManager. If the output buffer for task
+  /// is not found, prepares the Result object with completed flags set to false
+  /// and notifies the future.
+  ///
+  /// NOTE: This method is made public for unit testing purpose only.
```

Comment:
Not related to this change, as such, but this is a bad practice.
Like anything solely "needed for testing".
It makes code unnecessary complicated and flaky.

If we have to, we should have created a public test_bla() method, from which we call this.

Reply:
Update to testingGetDataForResultRequests based on Velox coding style: https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions